02_01_Advanced_Movement:_Microgravity.txt

```diff
@@ -0,0 +1,161 @@
+==============================
+Movement in Microgravity
+==============================
+
+===== States of Motion =====
+
+	In the absence of gravity, natural and artificial, movement changes 
+fundamentally. In Zero-G environments, characters have one of two states at all 
+times: Anchored or Drifting. 
+
+Anchored:
+	A character is Anchored if he or she is adjacent to and at rest relative 
+to a firm surface that weighs at least 5 times as much as the character. 
+
+Drifting:
+	A character is Drifting whenever that character is not Anchored.
+
+===== Movement While Anchored =====
+
+	If a character is Anchored to a ferrous surface in a way that permits 
+movement, he or she may make movements actions along that surface as if on a 
+planetary surface, though accommodation may have to be made for non-flat surfaces. 
+Also keep in mind that a character may not have their full speed available, 
+depending on their method of Anchor. 
+
+===== Launching =====
+
+	An Anchored character may spend two actions to launch him- or herself away 
+from the Anchoring surface. Characters launching in this way enter the Drifting 
+state with their standard movement speed and a direction of their choice. 
+
+	Characters may also be launched by certain events, such as explosives going 
+off nearby or the Anchored object being struck by a ship weapon. In those cases, 
+the character has an optional reflex save based on the force of the event to avoid 
+being Launched – failure causes the character to be Launched directly away from the 
+incoming force as described below.
+
+Small explosives or Blunt Melee Strike: 1m per 2 actions and 40 Reflex per 30 points of damage
+Demolition explosives or Vehicle/Ship Weapon: 8m per 2 actions and 50 Reflex per 30 hull points of damage
+
+	A character may spend an Action to Brace for Impact, which grants Advantage
+on any Reflex saves to avoid being Launched until the character takes another Action.
+
+	In the case that the Anchoring surface is a capital ship, station, or 
+similarly high-inertia object, characters are not Launched by strikes against the 
+surface that are more than 50m away except in truly calamitous circumstances, at 
+the DM’s discretion.
+
+	In the case that a character would be Launched toward an adjacent solid 
+object, such as the surface they were Anchored to, that character may make a DC 120 
+reflex save to catch themselves (Anchoring equipment may provide bonuses), in which 
+case they become Anchored again but gain the prone state. Failure causes the character 
+to uncontrolled Bounce. If the Launching event was not a surprise, the character may 
```

Comment:
Non blocking: "failure causes the character to bounce uncontrollably" flows a little better I think.